recipe/meta.yaml

```diff
@@ -1,21 +1,22 @@
-{% set version = "4.0.0" %}
+{% set version = "4.1.0" %}
 
 package:
   name: bleach
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/b/bleach/bleach-{{ version }}.tar.gz
-  sha256: ffa9221c6ac29399cc50fcc33473366edd0cf8d5e2cbbbb63296dc327fb67cc8
+  sha256: 0900d8b37eba61a802ee40ac0061f8c2b5dee29c1927dd1d233e075ebf5a71da
 
 build:
   number: 0
   noarch: python
   script: {{ PYTHON }} -m pip install . -vvv
+  skip: true  # [py<36]
```

Comment:
Skips should not be necessary for `noarch` Python packages.

Reply:
I am not sure how do we want to approach the skip section, I though that we wanted to specify the python versions in this particular case:

https://github.com/AnacondaRecipes/bleach-feedstock/blob/update_to_4.1.0/recipe/meta.yaml#L15